picATrip/urls.py

```diff
@@ -18,14 +18,22 @@
 from django.urls import path, include
 from django.conf import settings
 from django.conf.urls.static import static
-from pickATrip_django_apps import views
+from django.views.generic import TemplateView
 from users import views as user_views
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
-    path('', views.homepage, name='homepage'),
-    path('about/', views.about, name='about'),
+    path(
+        '',
+        TemplateView.as_view(template_name="pickATrip_django_apps/homepage.html"),
+        name='homepage',
+    ),
+    path(
+        'about/',
+        TemplateView.as_view(template_name="pickATrip_django_apps/about.html"),
+        name='about',
+    ),
```

Comment:
Those changes are the reason fails on tests. if we use Template.as_view I need to change the test because Template.as_view returns a different function on memory and python refers to this as different functions.
@maayanhd There is a reason you changed that?

Reply:
Actually, I did this at first because I thought it would be a better way, but I couldn't find if this is the case and haven't think of the fact it might fail future tests, thanks for clarifying.